doltcli/dolt.py

```diff
@@ -712,18 +712,35 @@ def branch(
         delete: bool = False,
         copy: bool = False,
         move: bool = False,
+        remote: bool = False,
+        all: bool = False,
         **kwargs,
     ):
         """
-        Checkout, create, delete, move, or copy, a branch. Only
-        :param branch_name:
-        :param start_point:
-        :param new_branch:
-        :param force:
-        :param delete:
-        :param copy:
-        :param move:
-        :return:
+        List all branches and optionally checkout, create, delete, move, or copy, a branch.
+
+        If <branch_name> is None, existing branches are listed, including remotely tracked branches
+        if 'remote' or 'all' are set. If <branch_name> is provided, a new branch is created, checked
+        our, deleted, moved or copied.
```

Comment:
let's decide if we want to use `''` or `<>` to denote variables. Personally I'd prefer ``

Reply:
Makes sense, let's go for ``. Reason for the mix was me copying directly from their online cli docs: https://docs.dolthub.com/cli-reference/cli#dolt-branch